src/sidebar/components/ShareDialog/test/ExportAnnotations-test.js

```diff
@@ -81,6 +81,11 @@ describe('ExportAnnotations', () => {
     $imports.$restore();
   });
 
+  const waitForSelect = async (wrapper, testId) => {
+    const selects = await waitForElement(wrapper, SelectNext);
```

Comment:
There is a potential hazard here that this could return too early if there is a `SelectNext` present, but it is the wrong one. I think `waitForElement(`[data-testid="${testId}"]`) will work and resolve this. I can't remember whether Enzyme allows attribute selectors without an element type.

Reply:
Good catch

Reply:
Enzyme works when passing only the attribute selector